src/Tempest/Console/ConsoleApplication.php

```diff
@@ -25,11 +26,13 @@ public function __construct(
     ) {
     }
 
-    public static function boot(string $name = 'Tempest', ?AppConfig $appConfig = null): self
-    {
-        $root = $appConfig->root ?? getcwd();
-        $appConfig ??= new AppConfig(root: $root);
-        $container = Tempest::boot($root, $appConfig);
+    /** @param \Tempest\Core\DiscoveryLocation[] $discoveryLocations */
+    public static function boot(
```

Comment:
TODO: we cannot enable/disable discovery cache with .env now anymore. We need to find a way to make that possible from with the kernel